tools/cluster.py

```diff
@@ -46,7 +46,7 @@
 default_ami = None # will be decided based on spark-ec2 list
 default_master_ami = None
 default_env = 'dev'
-default_spark_version = 'https://circle-artifacts.com/gh/chaordic/spark/3/artifacts/0/tmp/circle-artifacts.zAWvGZt/spark-1.2.2-SNAPSHOT-bin-1.0.4.tgz'
+default_spark_version = 'http://www.us.apache.org/dist/spark/spark-1.3.0/spark-1.3.0-bin-cdh4.tgz'
```

Comment:
default_spark_version = '1.3.0'

No need to add URL. 

Unless spark-ec2 has no support to spark-1.3.0 cdh4 based.